docs/managed-services/mariadb-managed-service.md

```diff
@@ -0,0 +1,150 @@
+---
+id: mariadb-managed-service
+title: MariaDB + phpMyAdmin
+sidebar_label: MariaDB + phpMyAdmin
+description: 'MariaDB یکی از محبوب‌ترین RDBMS‌های امروزی است که طرفداران زیادی در سرتاسر دنیا دارد؛ به همین دلیل MariaDB به عنوان اولین managed-service به فندق اضافه شد.'
+keywords:
+  - "سکوی ابری"
+  - "سرویس‌های مدیریت شده"
+  - سرویس
+  - داکر
+  - mariadb
+  - sql
+  - دیتابیس
+  - database
+  - "managed services"
+  - "سکوی ابری فندق"
+  - "زیرساخت ابری"
+image: /img/docs/thumbnails/managed-services/mysql-phpmyadmin-managed-service-thumbnail.png
+---
+
+![MySQL + PHPMyAdmin](/img/docs/mysql-phpmyadmin.svg "MySQL + PHPMyAdmin")
+
+MariaDB یکی از محبوب ترین  RDMBS های open source است که طرفداران خیلی زیادی دارد. <br/>
+در جدول زیر تفاوت های عمده ای که بین MariaDB و MySQL وجود دارد را مشاهده می فرمایید:
+
+|پارامتر|MariaDB|MySQL|
```

Comment:
Translations are not acceptable and some translations are out of user understanding scope!

Reply:
I try to translate them in better way, please check again